src/ImageSharp/Processing/Processors/Transforms/RotateInterpolationProcessor.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+using SixLabors.ImageSharp.Helpers;
+using SixLabors.ImageSharp.Memory;
+using SixLabors.ImageSharp.PixelFormats;
+using SixLabors.Primitives;
+
+namespace SixLabors.ImageSharp.Processing.Processors
+{
+    internal class RotateInterpolationProcessor<TPixel> : RotateProcessor<TPixel>
+        where TPixel : struct, IPixel<TPixel>
+    {
+        /// <inheritdoc/>
+        protected override void OnApply(ImageFrame<TPixel> source, Rectangle sourceRectangle, Configuration configuration)
+        {
+            if (this.OptimizedApply(source, configuration))
+            {
+                return;
+            }
+
+            int height = this.CanvasRectangle.Height;
+            int width = this.CanvasRectangle.Width;
+            Matrix3x2 matrix = this.GetCenteredMatrix(source, this.ProcessMatrix);
+            Rectangle sourceBounds = source.Bounds();
+
+            using (var targetPixels = new PixelAccessor<TPixel>(width, height))
+            {
+                Parallel.For(
+                    0,
+                    height,
+                    new ParallelOptions { MaxDegreeOfParallelism = 1 }, //configuration.ParallelOptions,
```

Comment:
If it doesn't work in parallel there's little point using `Parallel.For`

Reply:
Temporary commented out to aid debugging... It works fine in parallel.

Reply:
👍